docs/UserGuide.md

```diff
@@ -1,192 +1,69 @@
----
-layout: page
-title: User Guide
----
+# uMessage User Guide
 
-AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
+*uMessage* is an app that helps university students manage contacts, academics and CCAs that communicate and hold meetings on various platforms. It is optimized for keyboard users.
 
-* Table of Contents
-{:toc}
+## Feature List (v1.2)
 
---------------------------------------------------------------------------------------------------------------------
+### Add Contact: `add`
 
-## Quick start
+Add communication channels under contact (e.g. telegram, slack)
 
-1. Ensure you have Java `11` or above installed in your Computer.
+Adding a person: add
 
-1. Download the latest `addressbook.jar` from [here](https://github.com/se-edu/addressbook-level3/releases).
+**Format:** `add n/NAME \[d/{Social Media Platform} {ID} d/{Social Media Platform} {ID}\] \[t/cs2103\]...`
```

Comment:
perhaps we can delimit social media platform and id like so:
`d/Google Hangouts, cheehongw@gmail.com d/Telegram, @chwong1`

It would make sense to use commas since this is like a key-value pair, and easy to implement (`.split(",")`). 
Can anyone think of edge cases where there are commas in `social media platform` or `id`?